script/sync-ghes/settings.json

```diff
@@ -16,5 +16,15 @@
     "actions/starter-workflows",
     "actions/upload-artifact",
     "actions/upload-release-asset"
+  ],
+  "partners": [
+    "Alibaba Cloud",
```

Comment:
Does this mean we need a template validation which compares the creator field in properties.json to this list? Not needed for this PR, but food for thought.

Reply:
the first category listed is considered the "primary category" and hence keeping "Dockerfile" first in the list to maintain the status quo . [Link](https://github.com/github/github/blob/1bb860fea968de947cb4b740aab65f67404a01c9/app/view_models/actions/starter_workflows_view.rb#L156)